app/src/main/res/drawable/edit_text_background.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:drawable="@drawable/edit_text_background_shadow" android:state_focused="true" />
+  <item android:drawable="@drawable/edit_text_background_shadow" android:state_pressed="true" />
+  <item android:drawable="@drawable/edit_text_background_border" />
+  //means normal
```

Comment:
Remove this comment

Reply:
shifted it to right of tag for understanding normal tag